docker-compose.yml

```diff
@@ -95,7 +97,3 @@ services:
   redis:
     image: redis:latest
 
-networks:
-  default:
-    external:
-      name: docker_default # should match insights-upload's network name
```

Comment:
Without this defined as an external network, we won't be able to access kafka, defined in its own docker-compose, by its service name (i.e. `kafka`). https://github.com/RedHatInsights/insights-ingress-go/blob/master/docker-compose.yml